PostgresAssignment/src/database/index.ts

```diff
@@ -0,0 +1,22 @@
+const { Pool } = require("pg");
+import { dbName, dbHost, dbPort, dbPassword, dbUser } from "../config/config";
+
+const pool = new Pool({
+	port: 5432,
```

Comment:
please use config to load all the configuration. ref : https://github.com/adityajawade93/freshers-june2021/tree/main/tarun_kumar/PostgreAssignment